src/main/java/RetriveTimeline.java

```diff
@@ -0,0 +1,15 @@
+import twitter4j.*;
+
+import java.util.List;
+
+public class RetriveTimeline {
+
+
+    public static void getTimeLine() throws TwitterException {
+        Twitter twitter = TwitterFactory.getSingleton();
+        Paging page = new Paging (1, 10); //page number, number per page
+        List<Status> statuses = twitter.getUserTimeline("SonuSood",page);
```

Comment:
What if I want my home feed to be displayed?

Reply:
yea, then i will add a method for getHomeTimeline()